scim2/scim2_connector.bal

```diff
@@ -17,246 +17,247 @@
 //
 import ballerina/http;
 import ballerina/mime;
+import ballerina/io;
 
 # Object for SCIM2 endpoint.
-# + baseUrl - Base URL of the REST API
-# + httpClient - HTTP client endpoint
-public type ScimConnector object {
+#
+# + baseUrl - The base URL of the SCIM2 API
+# + scimClient - HTTP client endpoint
+public type ScimConnector client object {
```

Comment:
Shall we remove this `.bal` file after moving the implementation into `scim2_endpoint.bal`.
Please refer https://github.com/wso2-ballerina/module-twitter/pull/74